scripts/update_supported_languages_and_locales.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import os
+from tempfile import TemporaryFile
+
+from dateparser.data.languages_info import language_locale_dict
+
+
+def to_string(data):
+    result = ''
+    language_column_width = 18
+    for language in sorted(data):
+        result += language
+        locales = data[language]
+        if locales:
+            result += ' ' * (language_column_width - len(language))
+            result += ', '.join(
+                "'{}'".format(locale) for locale in sorted(locales)
+            )
+        result += '\n'
+    return result
+
+
+def main():
+    readme_path = os.path.join(os.path.dirname(__file__), '..', 'README.rst')
+    new_data = to_string(language_locale_dict)
+    temporary_file = TemporaryFile('w+')
+    with open(readme_path) as readme_file:
+        delimiter = '============    ================================================================\n'
```

Comment:
what about refactoring this to something like this?:
```suggestion
        delimiter = '='*12+'    '+'='*64+'\n'
```